tools/poi_osm_import/insert_fw_core_from_osm.sql

```diff
@@ -1,6 +1,6 @@
 ﻿INSERT INTO fw_core(
-            uuid, osm_id, timestamp, category, location, source_name, source_website, source_license)
-    SELECT uuid_generate_v4(), osm_id, 1410777181, amenity, Geography(ST_Transform(way,4326)), 'OpenStreetMap', 'http://www.openstreetmap.org', 'http://www.openstreetmap.org/copyright'
+            uuid, osm_id, timestamp, categories, location, source_name, source_website, source_license)
+    SELECT uuid_generate_v4(), osm_id, 1410777181, '{amenity}', Geography(ST_Transform(way,4326)), 'OpenStreetMap', 'http://www.openstreetmap.org', 'http://www.openstreetmap.org/copyright'
```

Comment:
<code>ARRAY[amenity]</code> seems to work better than <code>'{amenity}'</code>. The latter causes the string value "amenity" to be stored in the array element.